### DISCUSSION TITLE
Replace the Menu Bar for program windows (Alt+Space)

### DISCUSSION THREAD

**Discussion by griseouslight** [moudey/Shell#118] [Ideas]
I would like to know if it would be possible to add the feature to replace the Alt+Space menu for programs:
![image](https://user-images.githubusercontent.com/9774133/213584874-a5f2cc14-99a7-4127-88d1-d534f0e9ce08.png)

I would like to point out that it already does this for the taskbar specifically if you focus on it and press Alt+Space:
![image](https://user-images.githubusercontent.com/9774133/213584974-54f65492-56c8-493f-ae2a-383b97cc9535.png)

**Comment by moudey** [comment]
Please refer to [#181](https://github.com/moudey/Shell/discussions/181#discussioncomment-5355418)
